### PR TITLE
web/rest: reviewed API endpoints

### PR DESCRIPTION
**Pull request by mmadariaga**
- Remove deprecated API endpoints
- Add missing entities on Brand API:
   - FixedCostsRelInvoiceScheduler (And fixed database table definition)
   - InvoiceScheduler
- Add missing entities on Platform API:
  - Rptengine

#### Type Of Change 
- [ ] Small bug fix
- [X] New feature or enhancement
- [ ] Breaking change

#### Checklist:
- [X] Commits are named and have tag following [commit rules](https://github.com/irontec/ivozprovider/blob/bleeding/doc/dev/en/commits.md)
- [X] Commits are split per component (schema, web/admin, kamusers, agis, ..)
- [X] Changes have been tested locally
- [ ] Fixes an existing issue (Fixes #XXXX)

### CODE REVIEW

**Comment by ironArt3mis** [comment]
Rebase completed.

File: web/rest/brand/features/provider/fixedCostsRelInvoiceScheduler/putFixedCostsRelInvoiceScheduler.feature [hunk 1]
```diff
@@ -0,0 +1,44 @@
+Feature: Update fixed costs rel invoice schedulers
+  In order to manage fixed costs rel invoice schedulers
+  As a brand admin
+  I need to be able to update them through the API.
+
+  @createSchema
+  Scenario: Update a fixed cost rel invoice
```
Comment (Kaian) [inline web/rest/brand/features/provider/fixedCostsRelInvoiceScheduler/putFixedCostsRelInvoiceScheduler.feature]: Wrong description here

File: web/rest/brand/features/provider/fixedCostsRelInvoiceScheduler/removeFixedCostsRelInvoiceScheduler.feature [hunk 2]
```diff
@@ -0,0 +1,12 @@
+Feature: Manage fixed costs rel invoice schedulers
+  In order to manage fixed costs rel invoice schedulers
+  As a brand admin
+  I need to be able to delete them through the API.
+
+  @createSchema
+  Scenario: Remove a fixed cost rel invoice
```
Comment (Kaian) [inline web/rest/brand/features/provider/fixedCostsRelInvoiceScheduler/removeFixedCostsRelInvoiceScheduler.feature]: Wrong description

File: web/rest/brand/features/provider/invoiceScheduler/getInvoiceScheduler.feature [hunk 3]
```diff
@@ -0,0 +1,54 @@
+Feature: Retrieveinvoice scheduler
```
Comment (Kaian) [inline web/rest/brand/features/provider/invoiceScheduler/getInvoiceScheduler.feature]: missing space everywhere invoice is mentioned

File: web/rest/brand/features/provider/invoiceScheduler/postInvoiceScheduler.feature [hunk 4]
```diff
@@ -0,0 +1,90 @@
+  Feature: Createinvoice scheduler
```
Comment (Kaian) [inline web/rest/brand/features/provider/invoiceScheduler/postInvoiceScheduler.feature]: missing space everywhere invoice is mentioned

File: web/rest/brand/features/provider/invoiceScheduler/putInvoiceScheduler.feature [hunk 5]
```diff
@@ -0,0 +1,48 @@
+Feature: Updateinvoice scheduler
```
Comment (Kaian) [inline web/rest/brand/features/provider/invoiceScheduler/putInvoiceScheduler.feature]: missing space everywhere invoice is mentioned

File: web/rest/brand/features/provider/invoiceScheduler/removeInvoiceScheduler.feature [hunk 6]
```diff
@@ -1,12 +1,12 @@
-Feature: Manage users cdrs
-  In order to manage users cdrs
+Feature: Manageinvoice scheduler
```
Comment (Kaian) [inline web/rest/brand/features/provider/invoiceScheduler/removeInvoiceScheduler.feature]: missing space everywhere invoice is mentioned

**Comment by ironArt3mis** [comment]
Rebase completed.

**Pull request merged**